tests/lax_numpy_test.py

```diff
@@ -138,6 +138,7 @@ def op_record(name, nargs, dtypes, shapes, rng, diff_modes, test_name=None,
     op_record("expm1", 1, number_dtypes, all_shapes, jtu.rand_small_positive(), []),
     op_record("fix", 1, float_dtypes, all_shapes, jtu.rand_default(), []),
     op_record("floor_divide", 2, number_dtypes, all_shapes, jtu.rand_nonzero(), ["rev"]),
+    op_record("frexp", 1, default_dtypes, all_shapes, jtu.rand_default(), ["rev"]),
```

Comment:
Can you please:
(a) test a larger range of values? I believe `rand_default()` returns normally-distributed random numbers with a mean of zero and a standard deviation of 1; for this function in particular I would expect it to be useful to check a range of exponent values.
(b) test inf and nan values? I believe the current implementation does the wrong thing for inf.

(You might need to extend or refactor one of the random generators in `test_util.py`.)

Reply:
Sorry for the late response. Sure. Will do that.

By the way, I wonder how to enable float64 for a single test case. I could enable it via `config.update("jax_enable_x64", True)` but it seems that it will affect all testcases. In the case of `ldexp`, because the max value for float32 and float64 is different, for very large exponent, one will return inf, and the other will return a real value. Therefore, I'd like to test some values with float64.

Reply:
JAX's unit tests are run twice on Travis-CI: with and without float64 enabled. You can simply  check if `FLAGS.jax_enable_x64` is true to known if float64 is enabled (and if some test specifically needs float64 enabled or not, feel free to skip it otherwise)

Reply:
Thank you! Will update it soon!